src/components/modules/header/Header.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { css } from '@emotion/core';
+import { from, until } from '@guardian/src-foundations/mq';
+import { brandAlt, brandText } from '@guardian/src-foundations';
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { LinkButton, buttonReaderRevenueBrand } from '@guardian/src-button';
+import { ThemeProvider } from 'emotion-theming';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { HeaderProps } from '../../../types/HeaderTypes';
+import { addRegionIdAndTrackingParamsToSupportUrl } from '../../../lib/tracking';
+
+const messageStyles = (isThankYouMessage: boolean) => css`
+    color: ${brandAlt[400]};
+    ${headline.xxsmall({ fontWeight: 'bold' })};
+    padding-top: 3px;
+    margin-bottom: 3px;
+
+    ${from.desktop} {
+        ${headline.xsmall({ fontWeight: 'bold' })}
+    }
+
+    ${from.leftCol} {
+        ${isThankYouMessage
+            ? headline.small({ fontWeight: 'bold' })
+            : headline.medium({ fontWeight: 'bold' })}
+    }
+`;
+
+const linkStyles = css`
+    height: 32px;
+    min-height: 32px;
+    ${textSans.small({ fontWeight: 'bold' })};
+    border-radius: 16px;
+    padding: 0 12px 0 12px;
+    line-height: 18px;
+    margin-right: 10px;
+    margin-bottom: 6px;
+
+    svg {
+        width: 24px;
+    }
+`;
+
+const hiddenUntilTablet = css`
+    ${until.tablet} {
+        display: none;
+    }
+`;
+
+const hiddenUntilMobileMedium = css`
+    ${until.mobileMedium} {
+        display: none;
+    }
+`;
+
+const hiddenFromMobileMedium = css`
+    ${from.mobileMedium} {
+        display: none;
+    }
+`;
```

Comment:
just a suggestion - could potentially use the `Hide` component from source for this

Reply:
yes! I forgot about that